packages/diagrams/frontend/sirius-components-diagrams/src/sprotty/views/EdgeView.tsx

```diff
@@ -293,6 +296,33 @@ export class EdgeView extends PolylineEdgeView {
     return <circle transform={transform} r={4 + strokeWidth} style={styleObject} />;
   }
 
+  buildCrossedCircleArrow(p1, p2, styleObject, strokeWidth) {
```

Comment:
`buildCrossedCircleArrow(p1: Point, p2: Point, styleObject, strokeWidth: number)`

Reply:
Other methods `buildxxxArrox` could also be typed